lib/index.ts

```diff
@@ -86,9 +94,25 @@ class PWMetrics {
         await sheets.appendResults(results.runs);
       }
     }
+
+    if (resultHasExpectationErrors && this.flags.expectations) {
+      throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
```

Comment:
Yes I think the CLI should catch this exception and continue going. 

But IMO it makes sense that the node module throws. (Consider anyone doing a `require('pwmetrics')`)..

I'm open to alternatives tho.